README.md

```diff
@@ -40,16 +40,24 @@ Grace period:                          [------------- 24 hours ------]
 npm install --save-dev @guardian/pan-domain-node
 ```
 
+### Setup
+The library requires AWS credential to read the public key files from the S3 bucket `pan-domain-auth-settings` in `Workflow` account.  Please ensure that the execution environment has AWS credential for an IAM profile that has read access to the bucket.
+
+Currently, the bucket has been configured to allow access from any IAM users under a (fairly broad) list of AWS accounts (such as composer, cms-fronts, mobile etc).  Please get in touch with Workflow and Collaboration team if you need to get the read permission on the `pan-domain-auth-settings` bucket for your AWS acount.
```

Comment:
It might be worth clarifying your description a bit. Technically we are using a bucket in the Workflow account. However, this package is publicly available and could be utilised by others who are using Panda in a non-guardian environment. Therefore we can't assume that consumers of this library are going to be using the Guardian bucket. 

To avoid this assumption, we could remove references to the specific account and bucket and just mention bucket / object access is required. We could also link to the Panda documentation if that would be useful: https://github.com/guardian/pan-domain-authentication.

Reply:
A very good point.  I've updated the readme to make it more general.

I've also added an argument to `PanDomainAuthentication` for the AWS profile name, which is used when the application is running locally.  Previously I assumed that it was Workflow.